### PR TITLE
Pull in some goodRatings from goodReads

### PR DESCRIPTION
**Pull request by LukeSlev**
the ratings currently look kinda weird on the layout page (not sure why) but the api calls are working

### CODE REVIEW

File: app/src/main/java/com/stl/skipthelibrary/Activities/ViewBookActivity.java [hunk 1]
```diff
@@ -395,7 +397,9 @@ private void fillBookDescriptionFields(){
         title_element.setText(book.getDescription().getTitle());
         author_element.setText(book.getDescription().getAuthor());
         rating_element.setMax(book.getRating().getMaxRating());
-        rating_element.setNumStars((int) Math.round(book.getRating().getAverageRating()));
+        Log.d(TAG, "fillBookDescriptionFields: rating "+book.getRating());
+        rating_element.setRating((float)book.getRating().getAverageRating());
```
Comment (gillianpierce) [inline app/src/main/java/com/stl/skipthelibrary/Activities/ViewBookActivity.java]:
```suggestion
rating_element.setNumStars(5);
        rating_element.setRating((float)book.getRating().getAverageRating());
```

Comment (LukeSlev) [reply app/src/main/java/com/stl/skipthelibrary/Activities/ViewBookActivity.java]: @gillianpierce so having this in the xml isn't equivalent to that¿ https://github.com/CMPUT301W19T09/SkipTheLibrary/pull/54/files#diff-3e5dab69d039442f636cec22cba9bc77R93

Comment (gillianpierce) [reply app/src/main/java/com/stl/skipthelibrary/Activities/ViewBookActivity.java]: oh yes I believe it is

**Comment by gillianpierce** [comment]
I rate this PR 5/5, but I rate it's author 0/5 cuz he's mean

**Pull request merged**